decidim-bulletin_board-js/src/trustee/trustee.js

```diff
@@ -72,7 +72,11 @@ export class Trustee {
    * @returns {Promise<Boolean>}
    */
   async *setupKeyCeremony() {
-    const message = await this.waitForNextLogEntryResult();
+    let message;
+    while (!message) {
+      message = await this.waitForNextLogEntryResult();
+    }
```

Comment:
As mentioned before, I am a bit worried about this but since we are dealing with the dummy schema I think is ok for now. We can revisit this code when we add the ElectionGuard wrapper 👍